### ISSUE TITLE
Pangres does something to the root logger that now everything identifies as pangres

### ISSUE THREAD

**Issue by KIC** [ThibTrip/pangres#20]
as soon you add pangres as your dependency  suddenly all log messages identify themself as "pangres"

i.e. a log message from a custom module not looks like this:
```
2020-10-10 10:10:14 | INFO     | pangres     | index_utils:get_pandas_object:123 - passed item was None
```

I would actually expect that my applications logging settings do not change

**Comment by ThibTrip** [comment]
Hello @KIC ! Yes that is awful sorry about that :crying_cat_face:. This is related to the issue #18 and should be fixed by my new PR (see #21).
In addition to the screenshot I posted in issue #18 I also tested your particular problem interactively:

# Before PR #21

![image](https://user-images.githubusercontent.com/40694343/95690228-72d04e00-0c16-11eb-9b4a-1fba5beff5aa.png)

# After PR #21

![image](https://user-images.githubusercontent.com/40694343/95690262-a612dd00-0c16-11eb-871d-7580fb90e797.png)

I will now merge the PR. Feel free to reopen the issue if something still isn't as expected. I took the liberty of creating a new version. It will be out soon.

**Comment by KIC** [comment]
That was quick! Thank you very much version 2.2.1 solves the problem.
